.cursor/rules/product_requirements_document.mdc

```diff
@@ -0,0 +1,113 @@
+---
+description: 
```

Comment:
You can add a description regards to PRD in this case, for examples: `PRD Rule`

Reply:
Atualizado